templates/galaxy/config/job_conf.yml

```diff
@@ -47,6 +47,17 @@ galaxy_jobconf:
         amqp_consumer_timeout: 2.0
         amqp_publish_retry: "true"
         amqp_publish_retry_max_retries: 60
+    - id: pulsar_sanjay_runner
+      load: galaxy.jobs.runners.pulsar:PulsarMQJobRunner
+      params:
+        amqp_url: "pyamqp://galaxy_sanjay_pulsar:{{ rabbitmq_password_galaxy_sanjay_pulsar }}@mq.galaxyproject.eu:5671//pulsar/galaxy_sanjay_pulsar?ssl=1"
```

Comment:
the key is missing in this PR, or did I missed it?

Reply:
My key is already in the vault (added it the last time when we were testing Pulsar)